src/partials/head-scripts.hbs

```diff
@@ -18,17 +18,17 @@
     data-modal-header-bg-color="#71808A"
     data-modal-title-color="#fff"
     data-modal-disclaimer-text-color="#10A4B3"
-    data-modal-image-width="34"
-    data-modal-image-height="42"
+    data-modal-image-width="40"
+    data-modal-image-height="40"
     data-modal-title-font-family="PP Telegraf"
     data-modal-title-font-size="30px"
     data-modal-title-font-weight="400"
     data-project-color="#000000"
     data-modal-disclaimer="This is an experimental custom LLM for answering technical questions about Hazelcast. Answers are based **only** on Hazelcast documentation and support sources, but may not be fully accurate so please use your best judgement. Don't forget to rate the answers and give feedback!"
     data-modal-example-questions="How do I get started with Kubernetes?,What are the benefits of Enterprise Edition?"
```

Comment:
Needs a workaround for the border - if less than 40 char so can you add this in?
```suggestion
    data-modal-example-questions="How do I get started using Kubernetes with Hazelcast?,What are the benefits of Enterprise Edition?"
```